packages/sources/coingecko-test/helpers/createOverrides.js

```diff
@@ -0,0 +1,44 @@
+const axios = require('axios')
+const fs = require('fs/promises')
+
+const main = async () => {
+  const overridesFilePath = './src/config/overrides.json'
+
+  const existingOverrides = JSON.parse(await fs.readFile(overridesFilePath, { encoding: 'utf8' }))[
+    'coingecko'
+  ]
+
+  const response = await axios.get(
+    'https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&page=10',
+  )
+
+  const mapped = response.data.reduce((acc, coin) => {
+    const symbol = coin['symbol'].toUpperCase()
+    if (!(symbol in acc)) {
+      acc[symbol] = coin['id']
+    } // Uncomment to add an array of duplicate symbols
```

Comment:
We should be careful if we're going from a non-unique symbol to the unique id, but certainly this is better than nothing!

Reply:
I agree, going to resolve this for now since it will work for our current use case for now